builder/proxmox/config.go

```diff
@@ -71,6 +71,7 @@ type nicConfig struct {
 	MACAddress string `mapstructure:"mac_address"`
 	Bridge     string `mapstructure:"bridge"`
 	VLANTag    string `mapstructure:"vlan_tag"`
+	Firewall   *bool  `mapstructure:"firewall"`
```

Comment:
Is it useful to separate "not set" and false, if false is the default?

Reply:
The idea behind that was to follow the pattern of logging a message when a default value was selected. Otherwise you can't tell if the user did not provide one or simply set it to false.

Shall I just remove the message and always default to false?

Reply:
I think in this case it is quite okay to not log it, since the default is not likely to break anything. Most of the others that are logged are things that could potentially cause a non-booting VM or similar.
Sounds okay?

Reply:
Sounds good, let me make the change 👷